src/Presentation/Nop.Web/Views/Product/_ProductSpecifications.cshtml

```diff
@@ -6,21 +6,21 @@
         <div class="title">
             <strong>@T("Products.Specs")</strong>
         </div>
-        <div class="table-wrapper">
-            <table class="data-table">
-                <colgroup>
-                    <col width="25%" />
-                    <col />
+        <div role="table" class="table-wrapper">
+            <table role="presentation" class="data-table">
+                <colgroup role="row">
+                    <col role="columnheader" width="25%" />
+                    <col role="columnheader" />
                 </colgroup>
                 <tbody>
                     @for (var i = 0; i < Model.Count; i++)
                     {
                         var item = Model[i];
-                        <tr @(i % 2 == 0 ? Html.Raw(" class=\"odd\"") : Html.Raw(" class=\"even\""))>
-                            <td class="spec-name">
+                        <tr @(i % 2 == 0 ? Html.Raw(" role=\"row\" class=\"odd\"") : Html.Raw(" role=\"row\" class=\"even\""))>
```

Comment:
Add role=row before the c# code to not duplicate it.

Reply:
Each row of the table should have the role=row attribute added, so it is necessary for it to be duplicated each time a row is created in the code.

Reply:
```suggestion
                        <tr role="row" @(i % 2 == 0 ? Html.Raw("class=\"odd\"") : Html.Raw("class=\"even\""))>
```

Reply:
Ken please see what I meant

Reply:
I see now :) Thank you for the clarification. I have committed the change.